src/geo/cartodb-layer-group.js

```diff
@@ -46,15 +46,19 @@ var CartoDBLayerGroup = Backbone.Model.extend({
     return !!this.get('urls');
   },
 
-  getTileURLTemplates: function () {
+  getTileURLTemplates: function (type) {
+    type = type || 'png';
```

Comment:
Yeah, I agree with that, but maybe this is not the site... I think that the URL don't have to be formed with the png, and then replace it. It should be well formed from the root.

What do you think?